system_tests/delayedinboxlong_test.go

```diff
@@ -51,12 +52,17 @@ func TestDelayInboxLong(t *testing.T) {
 			}
 			l1Txs = append(l1Txs, l1tx)
 		}
-		// adding multiple messages in the same AddLocal to get them in the same L1 block
-		errs := builder.L1.L1Backend.TxPool().AddLocals(l1Txs)
+		wrappedL1Txs := make([]*txpool.Transaction, 0, messagesPerAddLocal)
+		for _, tx := range l1Txs {
+			wrappedL1Txs = append(wrappedL1Txs, &txpool.Transaction{Tx: tx})
```

Comment:
you can wrap the transactions inside the same loop that created l1Txs, no need for an extra pass just for wrapping. Also, you can probably use wrappedTxs only and avoid l1Txs entirely.

Reply:
Good idea, I think I had just mindlessly copied the pattern I saw in geth. Fixed.